machine-learning/lambda-v2/README.md

```diff
@@ -33,6 +39,8 @@ These sample components provide the following configuration parameters that can
 The following configuration is applicable for both the `com.lambda.DLRImageClassification` and 
 `com.lambda.DLRObjectDetection` inference components. 
 
+Since these inference components are lambda components, we can set the following configuration values only during the deployment. More info on setting configuration during deployment can be found [here](https://docs.aws.amazon.com/greengrass/v2/developerguide/update-component-configurations.html)
+
 - **accessControl** (Optional) : The object that contains the authorization policy that allows the component to publish messages to the default notifications topic.
 
     Default:
```

Comment:
Should this say "Example" instead of "Default", because we say above that you can't set default configuration on Lambda components?

Reply:
I updated `accessControl` and `PublishResultsOnTopic` to Example instead of default. Rest of the config values are provided as default from the source code. I can update the content if you still have a different opinion on this.